src/tests/profileLookup.test.js

```diff
@@ -0,0 +1,14 @@
+const contacts = require("../challenges/profileLookup.js")
+const lookUpProfile = require("../challenges/profileLookup.js")
```

Comment:
This is almost surely not the correct way to import 2 things.  But also you don't need contacts anyway.